includes/lib/transifex-live-integration-prerender.php

```diff
@@ -158,8 +158,18 @@ function call_curl( $url ) {
 	function callback( $buffer ) {
 		global $wp;
 		$output = $buffer;
+
+		# Try to get language code from the Transifex-Lang header.
+		# This is used in the instance where subdomains have been configured
+		# outside Wordpress so it does not have any knowledge of other
+		# subdomains.
+		$lang = urlencode($_SERVER['HTTP_X_TRANSIFEX_LANG']);
+
 		$debug_html = '<!--' . "\n";
 		$page_url = home_url( $wp->request );
+		if ( !empty($lang) ) {
+			$page_url = str_replace('www', $lang, $page_url);
+		}
 		$page_url = rtrim( $page_url, '/' ) . '/';
```

Comment:
@SofiaMargariti any clue why we add a slash at the end here? Doesn't this mess up if the page has a query param?

Reply:
@kouk this doesn't make much sense. I think the intention was to add a slash in the beginning of the url so that we avoid something like this: `http://www.some-prerender.comhttp://www.foo.bar/`. In any case this has been handled in another PR so I removed this line.